services/skus/service.go

```diff
@@ -1174,6 +1176,37 @@ func (s *Service) verifyCredential(ctx context.Context, req credential, w http.R
 	return handlers.WrapError(nil, "Unknown credential type", http.StatusBadRequest)
 }
 
+// verifyIOSNotification - verify the developer notification from appstore
+func (s *Service) verifyIOSNotification(ctx context.Context, txInfo *appstore.JWSTransactionDecodedPayload, renewalInfo *appstore.JWSRenewalInfoDecodedPayload) error {
+	if txInfo == nil || renewalInfo == nil {
+		return errors.New("notification has no tx or renewal")
+	}
+
+	// lookup the order based on the token as externalID
+	o, err := s.Datastore.GetOrderByExternalID(txInfo.OriginalTransactionId)
```

Comment:
It seems that `GetOrderByExternalID` is using parametrised queries. Just wanted to double check if we need to sanitise `OriginalTransactionId` here

Reply:
added validation of that originial transaction id in this commit: https://github.com/brave-intl/bat-go/pull/1649/commits/e54c7768b5be2af0b57995d0c9c8424fb4b41158